.github/workflows/asset_dispatch.yml

```diff
@@ -0,0 +1,43 @@
+# Triggered on dispatching the action
+# Create a release an build
+
+name: Asset - on dispatch; build release and publish
+on:
+  workflow_dispatch:
+    inputs:
+      image_tag:
```

Comment:
`image_tag` is a hangover from the docker image tag, it should probably be `release_name`?